go/internal/feast/model/featureview.go

```diff
@@ -13,12 +13,13 @@ const (
 	DUMMY_ENTITY_VAL  = ""
 )
 
-var DUMMY_ENTITY types.Value = types.Value{Val: &types.Value_StringVal{StringVal: DUMMY_ENTITY_VAL}}
+var DUMMY_ENTITY_VALUE types.Value = types.Value{Val: &types.Value_StringVal{StringVal: DUMMY_ENTITY_VAL}}
 
 type FeatureView struct {
-	Base     *BaseFeatureView
-	Ttl      *durationpb.Duration
-	Entities []string
+	Base          *BaseFeatureView
+	Ttl           *durationpb.Duration
+	Entities      []string
```

Comment:
do I understand correctly that `Entities` consists of entity names and `EntityColumns` stores join keys?

Reply:
> do I understand correctly that Entities consists of entity names and EntityColumns stores join keys?

yes, so I'll change `Entities` -> `EntityNames`